Assessments/assessments.md

```diff
@@ -106,6 +114,14 @@ card DisplayQuestion, then: QuestionError do
     end
 end
 
+card ValidateAge when not isnumber(age) or age > 150, then: QuestionError do
```

Comment:
@HawkiesZA Is there need to have a lower limit or SxD is happy with all entries below 150

Reply:
This is as we did in onboarding.